app/src/main/java/io/github/androidacademyglobal/AcademyApplication.kt

```diff
@@ -15,5 +17,6 @@ class AcademyApplication : Application() {
             androidContext(this@AcademyApplication)
             modules(appModule)
         }
+        Napier.base(DebugAntilog())
```

Comment:
What about different logging for debug/release?

Reply:
I'm on it, thanks